addons/crm/models/crm_team.py

```diff
@@ -199,68 +199,34 @@ def _alias_get_creation_values(self):
-
-        If cron runs on an hour- or minute-based schedule minimum assignment
-        performed is equivalent to 0.2 workdays to avoid rounding issues.
-        Max assignment performed is for 30 days as it is better to run more
-        often than planning for more than one month. Assign process is best
-        designed to run every few hours (~4 times / day) or each few days.
+        The cron is designed to run at least once a day or more. 
 
         See ``CrmTeam.action_assign_leads()`` and its sub methods for more
         details about assign process.
 
-        :param float work_days: see ``CrmTeam.action_assign_leads()``;
         """
-        assign_cron = self.sudo().env.ref('crm.ir_cron_crm_lead_assign', raise_if_not_found=False)
-        if not work_days and assign_cron and assign_cron.active:
-            if assign_cron.interval_type == 'months':
-                work_days = 30  # maximum one month of work
-            elif assign_cron.interval_type == 'weeks':
-                work_days = min(30, assign_cron.interval_number * 7)  # max at 30 (better lead repartition)
-            elif assign_cron.interval_type == 'days':
-                work_days = min(30, assign_cron.interval_number * 1)  # max at 30 (better lead repartition)
-            elif assign_cron.interval_type == 'hours':
-                work_days = max(0.2, assign_cron.interval_number / 24)    # min at 0.2 to avoid small numbers issues
-            elif assign_cron.interval_type == 'minutes':
-                work_days = max(0.2, assign_cron.interval_number / 1440)    # min at 0.2 to avoid small numbers issues
-        work_days = work_days if work_days else 1  # avoid void values
         self.env['crm.team'].search([
             '&', '|', ('use_leads', '=', True), ('use_opportunities', '=', True),
             ('assignment_optout', '=', False)
-        ])._action_assign_leads(work_days=work_days)
+        ])._action_assign_leads(force_quota=False)
         return True
 
-    def action_assign_leads(self, work_days=1, log=True):
+    def action_assign_leads(self, log=True):
         """ Manual (direct) leads assignment. This method both
 
           * assigns leads to teams given by self;
           * assigns leads to salespersons belonging to self;
 
         See sub methods for more details about assign process.
 
-        :param float work_days: number of work days to consider when assigning leads
-          to teams or salespersons. We consider that Member.assignment_max (or
-          its equivalent on team model) targets 30 work days. We make a ratio
-          between expected number of work days and maximum assignment for those
-          30 days to know lead count to assign.
-
         :return action: a client notification giving some insights on assign
           process;
         """
-        teams_data, members_data = self._action_assign_leads(work_days=work_days)
+        teams_data, members_data = self._action_assign_leads(force_quota=True)
```

Comment:
I would remove the `force_quota` mechanism; there is no use case where we want to force it.